zebrad/src/components/sync.rs

```diff
@@ -465,8 +465,8 @@ where
         let tips = std::mem::take(&mut self.prospective_tips);
 
         let mut download_set = HashSet::new();
+        tracing::info!(?tip, "trying to extend chain tips");
         for tip in tips {
```

Comment:
Tips have two fields, and there's usually only one tip, so this log might be ok
```suggestion
        for tip in tips {
            tracing::info!(?tip, "trying to extend chain tips");
```